### ISSUE TITLE
check tools output "Installed,running,but not managed by VMWare" although open_vm_tools_ok is set

### ISSUE THREAD

**Issue by diLLec** [BaldMansMojo/check_vmware_esx#168]
If just one other bad state is detectd, the _VM ... Installed,running,but not managed by VMWare._ messages are outputted as well and may confuse the user.

For example:
```
WARNING: 21 VMs checked for VMWare Tools state, 0 are not OK.
1 Installed,running and current.
1 Installed,running,version is not current
19 Installed,running,but not managed by VMWare
VM ... Installed,running,but not managed by VMWare.
VM ... Installed,running,but not managed by VMWare.
VM ... Installed,running,but not managed by VMWare.
VM ... Installed,running,but not managed by VMWare.
VM ... Installed,running,but not managed by VMWare.
VM ... Installed,running,but not managed by VMWare.
VM ... Installed,running,but not managed by VMWare.
VM ... Installed,running,but not managed by VMWare.
VM ... Installed,running,but not managed by VMWare.
VM ... Installed,running,but not managed by VMWare.
VM ...Installed,running,but not managed by VMWare.
VM ... Installed,running,version is not current.
VM ... Installed,running,but not managed by VMWare.
```

The bad state, which is causing the issue is `VM ... Installed,running,version is not current.`

Problem could be solved here:
https://github.com/BaldMansMojo/check_vmware_esx/blob/37d222ec9a2b662eeb41fe1ae1cce49d0db6dac1/modules/dc_runtime_info.pm#L511

By changing the code as follows:
```
                                 if (defined($openvmtools))
                                    {
                                    $actual_state = 0;
                                    }
                                 else
                                    {
                                    $actual_state = 1;
                                    $tools_out = $tools_out . "VM " . $vm->name . " ($openvmtools) Installed,running,but not managed by VMWare. " . $multiline;
                                    }
```

**Comment by BaldMansMojo** [comment]
This had been fixed by the option above.
